### PR TITLE
[Fix] Wrong page title for group edit page

### PR DESCRIPTION
**Pull request by kayathiri4**
### Related Issues
- https://github.com/wso2/product-is/issues/17895

### Related PRs
- None

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: .changeset/breezy-cows-change.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/react-components": patch
+"@wso2is/theme": patch
```
Comment (pavinduLakshan) [inline .changeset/breezy-cows-change.md:3]: Please include all the dependent packages as well in the changeset. You can view the dependents for each module by running `npx nx graph` in the project root

File: .changeset/fast-radios-sneeze.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/i18n": patch
+---
+
```
Comment (pavinduLakshan) [inline .changeset/fast-radios-sneeze.md:4]: Any specific reason for multiple changesets?

Comment (kayathiri4) [reply .changeset/fast-radios-sneeze.md:4]: To cover a previous PR

File: .changeset/breezy-cows-change.md (hunk 1 above)
Comment (kayathiri4) [reply .changeset/breezy-cows-change.md:3]: This is to cover the changes done with https://github.com/wso2/identity-apps/pull/4657.

**Review by Achintha444 — APPROVED** [review]

**Comment by kayathiri4** [comment]
Closing with https://github.com/wso2/identity-apps/pull/4681

**Pull request closed without merging**